dcc-portal-ui/app/scripts/advanced/js/advanced.js

```diff
@@ -503,6 +505,14 @@ angular.module('icgc.advanced.controllers', [
         SurvivalAnalysisLaunchService.launchSurvivalAnalysis(entityType, entityId, entitySymbol, filters);
       }
 
+      $rootScope.$on(FilterService.constants.FILTER_EVENTS.FILTER_UPDATE_EVENT, () => {
```

Comment:
Can we merge this with the 

```js
$scope.$on(_filterService.constants.FILTER_EVENTS.FILTER_UPDATE_EVENT,
```
[here](https://github.com/icgc-dcc/dcc-portal/blob/0d6a209eada0baaaa27ed7c2470ec21e8d0cf67f/dcc-portal-ui/app/scripts/advanced/js/advanced.js#L269)?